packages/icons/README.md

```diff
@@ -30,6 +30,7 @@ icons.CloseIcon
 icons.ExclamationIcon
 icons.FilterIcon
 icons.OpenIcon
+icons.SubtractIcon
```

Comment:
My bad for not mentioning in kickoff but what do you think about the name `MinusIcon`? @ryanjdiercks @fjallkraken

Reply:
@chantaldegaust  used `subtract` here.
https://zeroheight.com/3ddd0f892/p/699b01-icons/b/315972

Reply:
Got it, then let's keep it consistent